docs/advanced/external-ip-remapping.md

```diff
@@ -0,0 +1,105 @@
+# External IP remapping
+
+You can use liqo to map external IPs and make them reachable from a peered cluster. You can configure the external IP remapping using the **IP** CRD.
+
+```{warning}
+This feature is available only if [network module](/advanced/manual-peering.md) is enabled.
+```
+
+Check the figure below to understand how the external IP remapping works.
+We are going to make the **external host** reachable from **cluster 1**.
```

Comment:
Why cluster 1 needs to pass through cluster 2 to reach the external host? Isn't cluster 1 able to directly reach the host? I believe this is a point that should be clarified

Reply:
Yes, but it's something useless.